src/components/PeriodCard.vue

```diff
@@ -27,19 +22,35 @@
 <script lang="ts">
 import Vue from 'vue'
 import dayjs from 'dayjs'
+import minMax from 'dayjs/plugin/minMax'
 import ContentCard from '@/components/ContentCard.vue'
+dayjs.extend(minMax)
 
 export default Vue.extend({
   components: { ContentCard },
   props: {
+    period: {
+      type: Number,
+      default: 0
+    },
+    time: {
+      type: String,
+      default: ''
+    },
     classData: {
-      type: Object,
-      default: () => {}
+      type: Array,
```

Comment:
`import { classData } from '@/types/store/classData'` を追加したうえで
```suggestion
      type: Array as () => classData.LessonWithId[],
```
にするとエラーが解消されると思います

Reply:
ありがとうございます！すごいですね。エラー消えました！